Header header
uint16 lower_bound
uint16 upper_bound
